README.md

```diff
@@ -120,6 +126,27 @@ cli:
     #    cli-flatten-payload-max-poly-as-resource-prop-count: 8
     #    # max properties allowed from flatten of sub-class as param
     #    cli-flatten-payload-max-poly-as-param-prop-count: 8
+
+    # example for split-operation
+    # cli-directive:
+    #     - where:
+    #         group: OperationGroupName
+    #         op: CreateOrUpdate
+    #       split-operation-names:
+    #         - Create
+    #         - Update

```

Comment:
please also add this part into the cli-directive under doc folder

Reply:
Added. Thanks